draft-ietf-spice-sd-cwt.md

```diff
@@ -751,51 +709,53 @@ Reference: RFC XXXX
 
 IANA is requested to add the following entries to the CWT claims registry (https://www.iana.org/assignments/cwt/cwt.xhtml).
 
-### sd_alg
+<!-- https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-12#name-json-web-token-claims-regis -->
+
+### redacted_claim_keys
 
 The following completed registration template per RFC8392 is provided:
 
-Claim Name: sd_alg
-Claim Description: Hash algorithm used for selective disclosure
-JWT Claim Name: sd_alg
-Claim Key: TBD4 (request assignment 12)
-Claim Value Type(s): integer
+Claim Name: redacted_claim_keys
+Claim Description: Redacted Claim Keys in a map.
+JWT Claim Name: _sd
+Claim Key: TBD5 (request assignment 13)
+Claim Value Type(s): array of bstr
 Change Controller: IETF
 Specification Document(s): RFC XXXX
 
-### sd_hash
+### redacted_claim_element
 
 The following completed registration template per RFC8392 is provided:
 
-Claim Name: sd_hash
-Claim Description: Hash of encoded disclosed claims
-JWT Claim Name: sd_hash
-Claim Key: TBD3 (request assignment 11)
-Claim Value Type(s): bstr
+Claim Name: redacted_claim_element
+Claim Description: Redacted element of an array
+JWT Claim Name: ...
+Claim Key: TBD6 (request assignment 14)
```

Comment:
I don't think this is going to work as I described above.

Reply:
same issue https://github.com/ietf-wg-spice/draft-ietf-spice-sd-cwt/issues/19